src/conditional-flow/multiple-conditions.js

```diff
@@ -5,6 +5,10 @@ function isInRange (num, lower, upper) {
 
   // TODO: write code in this function body to pass the tests
 
+  if (num >= lower && num <= upper) {
```

Comment:
This could be simplified. `num >= lower && num <= upper` converts into a boolean value representing if the num is greater than or equal to lower and less than or equal to upper. So instead you could just return that value like so:
```js
return num >= lower && num <= upper
```